backoffice/src/wwwroot/js/site.js

```diff
@@ -63,6 +72,17 @@ $(function() {
             wreckFormFields.addClass("d-none");
         }
     }
+    
+    function toggleSalvorFields() {

```

Comment:
We probably want a new function toggleFormDisplay(IdField, formContainer)

or similar